Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Mech/Weapons/Gun/debug.yml

```
@@ -0,0 +1,98 @@
+- type: entity
+  id: WeaponMechDebugBallistic
+  parent: [ BaseMechWeaponRange, DebugMechEquipment ] # Debug equipment does have all whitelist tags.
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)